github.js

```diff
@@ -121,6 +121,10 @@
                   return cb(err);
                }
 
+               if (Object.prototype.toString.call(res) === '[object Object]') {
```

Comment:
`typeof` returns `'object'` for arrays, see: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof